cmd/mungedocs/mungedocs.go

```diff
@@ -29,6 +29,7 @@ import (
 )
 
 var (
+	verbose = flag.Bool("verbose", false, "On verification failure, emit pre-munge and post-munge versions.")
```

Comment:
> Seems like this should be in a separate PR.

Yeah, this was left in from earlier debugging attempts.  I'll move it to a different branch.